src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -63,33 +89,42 @@ static ColliderCylinderInit sCylinderInit = {
     { 30, 40, 0, { 0, 0, 0 } },
 };
 
-void func_80B13AA0(EnTa* this, EnTaActionFunc arg1, EnTaUnkFunc arg2) {
+void EnTa_SetActionFunctions(EnTa* this, EnTaActionFunc arg1, EnTaBlinkFunc arg2) {
     this->actionFunc = arg1;
-    this->unk_260 = arg2;
+    this->animFunc = arg2;
 }
 
-void func_80B13AAC(EnTa* this, PlayState* play) {
-    u16 faceReaction = Text_GetFaceReaction(play, 24);
+void EnTa_SetTextForTalkInLonLonHouse(EnTa* this, PlayState* play) {
+    u16 faceReaction = Text_GetFaceReaction(play, TALON_FACE_REACTION_SET);
 
-    if (GET_EVENTINF(EVENTINF_HORSES_0A)) {
-        if (GET_EVENTINF(EVENTINF_HORSES_08)) {
-            if (GET_ITEMGETINF(ITEMGETINF_02)) {
+    // Check if cucco game was just finished
+    if (GET_EVENTINF(EVENTINF_CUCCO_GAME_FINISHED)) {
+        // Check if the game was won
+        if (GET_EVENTINF(EVENTINF_CUCCO_GAME_WON)) {
+            if (GET_ITEMGETINF(ITEMGETINF_TALON_BOTTLE)) {
+                // "Why don't you come to work here, on this ranch"
                 this->actor.textId = 0x2088;
```

Comment:
We had such comments on text ids before (long ago), but it's not something we currently do anymore

I think in these cases (text ids in this file) comments like `// first time winning` would be good instead